src/masoniteorm/connections/decorators.py.49ec9b0f8ff47a3ba27e2805c82b56ea.tmp

```diff
@@ -0,0 +1,20 @@
+from config.database import db
+
+
+class Transaction(object):
+
+    def __init__(self, connection="default"):
+        self.connection = connection
+
+    def __call__(self, fn, *args, **kwargs):
+        # before function
+        db.begin_transaction(self.connection)
+        try:
+            result = fn(*args, **kwargs)
```

Comment:
Here you are calling fn() as a function and passing args to it. The result of the create_user() function is the User class. So right now this result variable is a user class.